### PR TITLE
HAWQ-1451. HAWQ state should be able to report the status of both RPS…

### PR DESCRIPTION
**Pull request by stanlyxiang**
… and standby RPS

### CODE REVIEW

File: tools/bin/hawqstate [hunk 1]
```diff
@@ -135,12 +138,14 @@ def show_brief_status(hawq_site, segment_list, standby_host):
...
-        hawq_acl_type = hawq_site.hawq_dict['hawq_acl_type']
-        logger.info("-   Current HAWQ acl type                          = %s" % hawq_acl_type)
-        if hawq_acl_type == "ranger":
-            hawq_rps_status = check_rps_status()
-            logger.info("-   HAWQ Ranger plugin service state               = %s" % hawq_rps_status)
+    logger.info("-----------------------------------------------------")
+    logger.info("-   Current HAWQ acl type                          = %s" % hawq_acl_type)
+    if hawq_acl_type == "ranger":
+        hawq_master_rps_status = check_rps_status('hawq_master_address_host')
+        logger.info("-   HAWQ master  Ranger plugin service state       = %s" % hawq_master_rps_status)
```
Comment (linwen) [inline tools/bin/hawqstate]: a redundant space between "master" and "Ranger"

**Comment by linwen** [comment]
+1

File: tools/bin/hawqstate (hunk 1 above)
Comment (stanlyxiang) [inline tools/bin/hawqstate]: fixed with new commit. Thanks

**Comment by stanlyxiang** [comment]
Merged into master. Close this PR.

**Pull request closed without merging**